inspirehep/dojson/hep/fields/bd70x75x.py

```diff
@@ -57,7 +57,13 @@ def thesis_supervisor2marc(self, key, value):
 def collaboration(self, key, value):
     """Added Entry-Corporate Name."""
     def get_value(value):
-        return value.get('g')
+        recid = ''
+        if '0' in value:
+            recid = int(value.get('0'))
```

Comment:
This will crash if there is no key `0` in value. You can do the `try...except` like above.